src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/LocationsReconciliationService.kt

```diff
@@ -90,23 +91,24 @@ class LocationsReconciliationService(
           allDpsIdsInNomis.contains((it.id.toString()))
         }
         .map { dpsRecord ->
+          val legacyRecord = locationsApiService.getLocation(dpsRecord.id.toString(), true)
```

Comment:
Now need to get each location in legacy format. I'll talk to Mike W about adding another endpoint that gets pages of locations in legacy so we can avoid this extra call